pkg/rtc/participant.go

```diff
@@ -1572,36 +1587,6 @@ func (p *ParticipantImpl) onSubscribedMaxQualityChange(trackID livekit.TrackID,
 		SubscribedCodecs:    subscribedQualities,
 	}
 
-	// send layer info about max subscription changes to telemetry
```

Comment:
drive-by as I was looking to synthesise this even on migration. Noticed that this is reading a bunch of stuff and synthesising `TrackInfo`. Just got it from track and passing it on as is.